### PR TITLE
always create a 32x32 image, even if application.current is null

### PR DESCRIPTION
**Pull request by rrelyea**
## Bug

Fixes: https://github.com/nuget/client.engineering/issues/603
Regression: no

## Fix

Details: tests sometimes run with application.current not null, sometimes not. make tests handle both cases. 

## Testing/Validation

Tests Added: no
Reason for not adding tests:  tweaking the new ones i had as part of feature work
Validation:  ran unit tests in VS. Then CI.

### CODE REVIEW

File: test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using System.IO;
 using System.Reflection;
 using System.Threading.Tasks;
+using System.Windows;
```
Comment (dominoFire) [inline test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs]: Do you need this namespace?

**Review by dominoFire — APPROVED** [review]

**Review by donnie-msft — APPROVED** [review]
Is the `Application.Current` sometimes being null flaky, or is it dependent on the test scenario whether it's loaded? If flaky, we should track an issue for understanding why

File: test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs (hunk 1 above)
Comment (rrelyea) [reply test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs]: nope... will remove. thx.

**Comment by rrelyea** [comment]
> Is the `Application.Current` sometimes being null flaky, or is it dependent on the test scenario whether it's loaded? If flaky, we should track an issue for understanding why

it is depending on whether other tests run before it or not, etc... i believe.

**Review by dtivel — APPROVED** [review]

**Review by erdembayar — APPROVED** [review]

**Pull request merged**